### PR TITLE
use new lg dictionary in export raw data

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Review https://github.com/AfricasVoices/Project-GPSDD/pull/35 first

### CODE REVIEW

File: export_weekly_advert_contacts.py [hunk 1]
```diff
@@ -80,14 +80,16 @@
...
-    for listening_group_csv in listening_group_csvs_names:
-        with open(f"{listening_group_data_dir}/{listening_group_csv}", "r", encoding='utf-8-sig') as f:
-            data = list(csv.DictReader(f))
-            log.info(
-                f"Loaded {len(data)} listening group participants from {listening_group_data_dir}/{listening_group_csv}")
-            for row in data:
-                uuids.add(row['avf-phone-uuid'])
-                listening_group_participants +=1
+    for k in pipeline_configuration.listening_group_csv_urls.keys():
+        for listening_group_csv_url in pipeline_configuration.listening_group_csv_urls[f'{k}']:
```
Comment (DanielMwendwa) [inline export_weekly_advert_contacts.py]:
```suggestion
        for listening_group_csv_url in pipeline_configuration.listening_group_csv_urls[k]:
```

File: export_weekly_advert_contacts.py [hunk 2]
```diff
@@ -80,14 +80,16 @@
...
+            listening_group_csv = listening_group_csv_url.split("/")[-1]
+            with open(f"{listening_group_data_dir}/{listening_group_csv}", "r", encoding='utf-8-sig') as f:
+                data = list(csv.DictReader(f))
+                log.info(
+                    f"Loaded {len(data)} listening group participants from {listening_group_data_dir}/{listening_group_csv}")
+                for row in data:
+                    uuids.add(row['avf-phone-uuid'])
+                    listening_group_participants +=1
```
Comment (DanielMwendwa) [inline export_weekly_advert_contacts.py]:
```suggestion
                    listening_group_participants += 1
```

File: export_weekly_advert_contacts.py [hunk 3]
```diff
@@ -80,14 +80,16 @@
...
     log.info(f'loaded {listening_group_participants} listening group participants')
```
Comment (DanielMwendwa) [inline export_weekly_advert_contacts.py]:
```suggestion
    log.info(f"loaded {listening_group_participants} listening group participants")
```

File: export_weekly_advert_contacts.py [hunk 4]
```diff
@@ -80,14 +80,16 @@
...
-                listening_group_participants +=1
+    for k in pipeline_configuration.listening_group_csv_urls.keys():
+        for listening_group_csv_url in pipeline_configuration.listening_group_csv_urls[f'{k}']:
+            listening_group_csv = listening_group_csv_url.split("/")[-1]
+            with open(f"{listening_group_data_dir}/{listening_group_csv}", "r", encoding='utf-8-sig') as f:
+                data = list(csv.DictReader(f))
+                log.info(
+                    f"Loaded {len(data)} listening group participants from {listening_group_data_dir}/{listening_group_csv}")
+                for row in data:
+                    uuids.add(row['avf-phone-uuid'])
```
Comment (DanielMwendwa) [inline export_weekly_advert_contacts.py]:
```suggestion
                    uuids.add(row["avf-phone-uuid"])
```

File: export_weekly_advert_contacts.py [hunk 5]
```diff
@@ -80,14 +80,16 @@
...
 
-    for listening_group_csv in listening_group_csvs_names:
-        with open(f"{listening_group_data_dir}/{listening_group_csv}", "r", encoding='utf-8-sig') as f:
-            data = list(csv.DictReader(f))
-            log.info(
-                f"Loaded {len(data)} listening group participants from {listening_group_data_dir}/{listening_group_csv}")
-            for row in data:
-                uuids.add(row['avf-phone-uuid'])
-                listening_group_participants +=1
+    for k in pipeline_configuration.listening_group_csv_urls.keys():
```
Comment (DanielMwendwa) [inline export_weekly_advert_contacts.py]: Can you replace 'k' with something descriptive?

**Review by DanielMwendwa — APPROVED** [review]

File: export_weekly_advert_contacts.py (hunk 1 above)
Comment (IsaackMwenda) [reply export_weekly_advert_contacts.py]: updated thanks.

File: export_weekly_advert_contacts.py (hunk 2 above)
Comment (IsaackMwenda) [reply export_weekly_advert_contacts.py]: updated thanks.

File: export_weekly_advert_contacts.py (hunk 3 above)
Comment (IsaackMwenda) [reply export_weekly_advert_contacts.py]: updated thanks.

File: export_weekly_advert_contacts.py (hunk 5 above)
Comment (IsaackMwenda) [reply export_weekly_advert_contacts.py]: updated to listening_group_type thanks.

**Pull request merged**